app/lib/features/link_room/types.dart

```diff
@@ -0,0 +1,5 @@
+// ChildRoomType configures the sub child type of the `Spaces`
+enum ChildRoomType {
+  chat,
+  space,
```

Comment:
removed the "recommneded" type as we consider all of them the same, it is really just a question of which listing provider we are using.